  <launch>
    <arg name="robot_name" default="iris" />
    <arg name="voxel_size" default="0.20" />
    <arg name="voxels_per_side" default="16" />
    <arg name="world_frame" default="map" />

      <node name="voxblox_node" pkg="voxblox_ros" type="esdf_server" output="screen" args="-alsologtostderr" clear_params="true">
        <remap from="pointcloud" to="/depth_camera/depth/points"/>
        <remap from="voxblox_node/esdf_map_out" to="esdf_map" />
        <remap from="voxblox_node/tsdf_map_out" to="tsdf_map" />
        <param name="tsdf_voxel_size" value="$(arg voxel_size)" />
        <param name="tsdf_voxels_per_side" value="$(arg voxels_per_side)" />
        <param name="publish_esdf_map" value="true" />
        <param name="publish_pointclouds" value="true" />
        <param name="use_tf_transforms" value="true" />
        <param name="update_mesh_every_n_sec" value="1.0" />
        <param name="clear_sphere_for_planning" value="true" />
        <param name="world_frame" value="$(arg world_frame)" />
      </node>
<!-- 
      <node name="my_voxblox_planner" pkg="voxblox_planner" type="my_voxblox_planner" output="screen" args="-alsologtostderr">
        <remap from="odometry" to="great_estimator/odometry" />
        <remap from="my_voxblox_planner/esdf_map_in" to="esdf_map" />
        <param name="tsdf_voxel_size" value="$(arg voxel_size)" />
        <param name="tsdf_voxels_per_side" value="$(arg voxels_per_side)" />
        <param name="update_mesh_every_n_sec" value="0.0" />
        <param name="world_frame" value="$(arg world_frame)" />
      </node> -->

  </launch>